data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/security/Credentials.java

```diff
@@ -25,6 +25,13 @@
    */
   String caCertificates();
 
+  /**
+   * Skip client auth.
+   *
+   * @return true if client auth should be skipped otherwise false
+   */
+  boolean skipUser();
```

Comment:
Not sure the name is really correct ... :thinking:

Reply:
`skipClientAuth`?